ui-trackoccupancydiagram/src/components/helpers/drawElements/drawOccupancyZonesTexts.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  OCCUPANCY_ZONE_START,
+  MINUTES_TEXT_OFFSET,
+  STATION_TEXT_OFFSET,
+  COLORS,
+} from '../../consts';
+import { drawText } from '../../utils';
+
+const BREAKPOINTS = {
+  medium: 24,
+  small: 4,
+};
+const STROKE_WIDTH = 4;
+const X_INITIAL_POSITION_OFFSET = 8;
+const Y_INITIAL_POSITION_OFFSET = 5;
+const Y_MEDIUM_POSITION_OFFSET = 14;
+
+const { WHITE_100, GREY_50, GREY_60, GREY_80 } = COLORS;
+
+export const drawOccupancyZonesTexts = ({
+  ctx,
+  zone,
+  arrivalTime,
+  departureTime,
+}: {
+  ctx: CanvasRenderingContext2D;
+  zone: {
+    arrivalTrainName: string;
+    arrivalTime: Date;
+    departureTime: Date;
+    originStation?: string;
+    destinationStation?: string;
+  };
+  arrivalTime: number;
+  departureTime: number;
+}) => {
+  const zoneOccupancyLength = departureTime - arrivalTime - STROKE_WIDTH;
+
+  const getBreakpoint = (breakpoint: keyof typeof BREAKPOINTS) => {
+    if (zoneOccupancyLength < BREAKPOINTS[breakpoint]) {
+      return true;
+    }
+    return false;
+  };
```

Comment:
i think you can simply `return zoneOccupancyLength < BREAKPOINTS[breakpoint]`

Reply:
and rename the function isBelowBreakpoint to clarify the meaning

Reply:
done